source/installation-guide/installing-wazuh-agent/solaris/solaris11/index.rst

```diff
@@ -0,0 +1,24 @@
+.. Copyright (C) 2019 Wazuh, Inc.
+
+.. _wazuh_agent_solaris11:
+
+Solaris 11
+==========
+
+The Wazuh agent is available for Solaris 11 i386 and Solaris 11 SPARC. The Wazuh agent can be installed via package or sources:
+
++---------------------------------------------------------------------------+-----------------------------------------------------------------------+
+| Type                                                                      | Description                                                           |
++===========================================================================+=======================================================================+
+| :ref:`Package installation <wazuh_agent_package_solaris11>`               | Install Wazuh agent Solaris 11 via package.                           |
```

Comment:
Lets unify "from package" or "via package" in all documents

Reply:
Done :+1: